contracts/OXO.sol

```diff
@@ -0,0 +1,90 @@
+pragma solidity 0.4.15;
+
+import "./OxoFactory.sol";
+
+contract OXO is OxoFactory{
+
+ 	event StepEvent(address player, uint fileld, State state);
+ 	event GameOverEvent(State state);
+	event TransferEvent(string _text, address _to, uint _value);
+
+	 function doStep(uint _field) stillCanPlay() onlyWaitThisPlayer() onlyEmptyCell(_field) public{
+	   Game memory game = gameById[getGameId()];
+		game.board[_field] = (msg.sender == game.playerOne)? uint(keccak256("x")) : uint(keccak256("o"));
+		State _state = game.state;
+		game.countOfMove ++ ;
+		if(game.countOfMove >= 5){
+		    _state = checkWinner(game, _field);
+			if(_state != State.WinnerPlayerOne && _state != State.WinnerPlayerTwo ){
+		    	_state = checkDraw(game);
+			}
+		}else{
+    		_state =  stateForNextPlayer(game);
+		}
+		StepEvent(msg.sender, _field, _state);
+		game.state = _state;
+		game.status = stateToString(_state);
+		game.next = (msg.sender == game.playerOne)? game.playerTwo : game.playerOne;
+		gameById[getGameId()] = game;
+    }
+
+    function checkDraw(Game _game) private returns(State){
```

Comment:
If you use the same number of indents throw the contract the universe will be benevolent with you

Reply:
:-)

Reply:
Strange
I did format code http://take.ms/5KoVe but when I look on GitHub I still have not right  number of indents